### ISSUE TITLE
colorize_reconstruction produces a reconstruction file same size, runs just few seconds

### ISSUE THREAD

**Issue by alkavan** [sweeneychris/TheiaSfM#221]
So here's the pipeline for my reconstructions:

```
create_calibration_file_from_exif \
-images=img/* \
-output_calibration_file=calibration
```

```
build_reconstruction \
-reconstruction_estimator=GLOBAL \
-calibration_file=calibration \
-descriptor=SIFT \
-feature_density=DENSE \
-images=img/* \
-matching_working_directory=matches/ \
-output_reconstruction=model \
-reconstruct_largest_connected_component=false \
-shared_calibration=true
-num_threads=4
```

```
colorize_reconstruction \
-image_directory=img/ \
-input_reconstruction_file=model-0 \
-num_threads=4 \
-output_reconstruction_file=model_color
```

So it seems that `colorize_reconstruction` produced the same reconstruction file... what does it mean? it also takes just a few seconds to run, and I'm assuming it does not actually do anything related to color processing?
Am I confusing or missing something here?
And on the same subject, it would be nice to have a basic README.md in the main repository with some basic pipeline examples, I only figured out what to do because I used OpenMVG and the `-help` flag ...

**Comment by sweeneychris** [comment]
As you can see from the command, you've provided an --output_reconstruction
flag where the colored reconstruction will write to. I don't know why you
assume that an app called colorize_reconstruction does nothing related to
color processing... obviously it does.

There is a ton of documentation online www.theia-sfm.org, every app is
documented inside the code, and (as documented) you can run any binary with
--helpshort to get the command line options. Please consider doing a bit of
investigation into the existing resources before claiming the apps do
nothing or more documentation is needed. I put a lot of work into
documenting the code and the website so please utilize those resources.

**Comment by charlescva** [comment]
Agreed. The documentation is very good as is the code base. I can concur
that be colorization code absolutely does work in the 0.8 release.
